pkg/api/image.go

```diff
@@ -128,3 +141,44 @@ func (h *Handler) CreateImage(w http.ResponseWriter, r *http.Request) {
 
 	w.Write([]byte(fmt.Sprintf("succeeded created image %s", createdImage.Name())))
 }
+
+// Deletes a block image from this cluster.
+// DELETE
+// /image?name=<imageName>&pool=<imagePool>
```

Comment:
We shouldnt provide the image name as query param. Query param is normally used for filtering/search. 
Ideally we would do something like `DELETE /image/{image_id}` and we would have some sort of datastore where we could get where the pool is. But this is part of #500.